pigar/parser.py

```diff
@@ -14,6 +14,18 @@
 from .helpers import parse_git_config, trim_suffix
 
 import nbformat
+try:
+    import IPython
+    # untested
+    # only support IPython >= 6.0.0
+    _transformer_manager = IPython.core.inputtransformer2.TransformerManager()
```

Comment:
`IPython` should be present in `setup.py`,

Reply:
Sadly, for the parser to works (my version), it requires IPython >= 6, whereas in colab, it's IPython 5. The problem, both versions have different Transformer API.

Reply:
I remove IPython from the `setup.py` because it's not the core for pigar. It's just for `pigar` on `IPython`. If you want to make all support for `IPython`, that's good, but sadly I can't help because I can't run the test (can't debug your codes, i don't know why).